lms/djangoapps/api_manager/management/commands/add_new_permissions.py

```diff
@@ -0,0 +1,61 @@
+from django.core.management.base import BaseCommand
+
+import uuid
+import json
+from django.core.exceptions import ObjectDoesNotExist
+
+from api_manager.models import GroupRelationship, GroupProfile
+from django.contrib.auth.models import User, Group
+from organizations.models import Organization, OrganizationGroupUser
+
+class Command(BaseCommand):
+    help = """Repair users that are assistants and observers on every course
+example:
+    manage.py lms add_new_permissions --settings={aws, devstack}
+"""
+
+    def handle(self, *args, **options):
+        
+        group_type = "permission"
+        original_group_name = "mcka_role_company_admin"
+        
+        try:
+            mcka_company_admin = Group.objects.get(name__icontains=original_group_name)
+            print "Found company admin group ("+original_group_name+"), id: " + str(mcka_company_admin.id)
+        
+        except ObjectDoesNotExist:
+            print "Creating new company admin group (mcka_role_company_admin)..."
+            mcka_company_admin = Group.objects.create(name=str(uuid.uuid4()))
+            mcka_company_admin.name = '{:04d}: {}'.format(mcka_company_admin.id, original_group_name)
```

Comment:
Use ``mcka_company_admin.name`` if you're using the name generated in the line above.